phys/module_mp_p3.F

```diff
@@ -354,37 +459,38 @@ SUBROUTINE p3_init(lookup_file_dir,nCat)
 !print*, '   Generating rain lookup-table ...'
 
  do i = 1,150              ! loop over lookup table values
-    initlamr = 1./((real(i)*2.)*1.e-6 + 250.e-6)
-
-! iterate to get mu_r
-! mu_r-lambda relationship is from Cao et al. (2008), eq. (7)
-
-! start with first guess, mu_r = 0
-
-    mu_r = 0.
-
-    do ii=1,50
-       lamr = initlamr*((mu_r+3.)*(mu_r+2.)*(mu_r+1.)/6.)**thrd
-
-! new estimate for mu_r based on lambda
-! set max lambda in formula for mu_r to 20 mm-1, so Cao et al.
-! formula is not extrapolated beyond Cao et al. data range
-       dum  = min(20.,lamr*1.e-3)
-       mu_r = max(0.,-0.0201*dum**2+0.902*dum-1.718)
-
-! if lambda is converged within 0.1%, then exit loop
-       if (ii.ge.2) then
-          if (abs((lamold-lamr)/lamr).lt.0.001) goto 111
-       end if
-
-       lamold = lamr
-
-    enddo
-
-111 continue
+! ! !     initlamr = 1./((real(i)*2.)*1.e-6 + 250.e-6)
```

Comment:
I get that this is contributed code. Is this commented out part used for a different model? Is it an old way to do something?